.github/workflows/release-it.yml

```diff
@@ -12,7 +12,7 @@ on:
         default: 'patch'
 jobs:
   release-it:
-    if: contains('["robertsLando","chrisns"]', github.actor)
+    if: contains(["robertsLando","chrisns"], github.actor)
```

Comment:
And now I'm not sure if that works. Either
```
if: contains("robertsLando,chrisns", github.actor)
```
which is probably unsafe if someone decides to call himself "robert", or with `fromJSON`:
https://github.community/t/passing-an-array-literal-to-contains-function-causes-syntax-error/17213/3

Reply:
Used the safer way: `    if: github.actor == 'robertsLando' || github.actor === 'chrisns'`